packages/ketcher-core/src/domain/entities/DrawingEntitiesManager.ts

```diff
@@ -441,16 +443,22 @@ export class DrawingEntitiesManager {
         });
         command.addOperation(monomerAddOperation);
         polymerBond.setSecondMonomer(monomer);
-        assert(previousMonomer.firstFreeAttachmentPoint);
-        assert(monomer.availableAttachmentPointForBondEnd);
-        previousMonomer.setBond(
-          previousMonomer.firstFreeAttachmentPoint,
-          polymerBond,
-        );
-        monomer.setBond(
-          monomer.availableAttachmentPointForBondEnd,
-          polymerBond,
-        );
+
+        // requirements are: Base(R1)-(R3)Sugar(R1)-(R2)Phosphate
+        const attPointStart = previousMonomer.R1AttachmentPoint;
+        let attPointEnd;
+
+        if (monomer instanceof Sugar) {
+          attPointEnd = monomer.isAttachmentPointUsed('R3') ? undefined : 'R3';
+        } else if (monomer instanceof Phosphate) {
+          attPointEnd = monomer.R2AttachmentPoint;
+        }
```

Comment:
I believe that the logic of should be inside of each type of monomer or inside bond. If no we will have such situation when user tries to make rna preset manually(R1 attachment point is used by default for both monomers):
<img width="578" alt="image" src="https://github.com/epam/ketcher/assets/14859362/925abf11-fd31-4bea-85f4-3b7702681810">
Let's discuss it with team on the meeting.